components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -37,13 +34,12 @@ trait Validator {
 @ConfiguredJsonCodec sealed trait ParameterValidator extends Validator
 
 //TODO: These validators should be moved to separated module
-
 case object MandatoryParameterValidator extends ParameterValidator {
 
-  override def isValid(paramName: String, expression: String, label: Option[String])(
+  override def isValid(paramName: String, value: Any, label: Option[String])(
       implicit nodeId: NodeId
   ): Validated[PartSubGraphCompilationError, Unit] =
-    if (StringUtils.isNotBlank(expression)) valid(()) else invalid(error(paramName, nodeId.id))
+    if (value != null) valid(()) else invalid(error(paramName, nodeId.id))
```

Comment:
I had a discussion with @mslabek about that behaviour and we agreed that we should change this:
- Let's `MandatoryParameterValidator` validate only that field is filled. It is not possible based only on value, we need to add expression: `pl.touk.nussknacker.engine.graph.expression.Expression` next to it.
- Let's add `NotNullParameterValidator` that would check if value is not null. Caution - it should avoid checking this when an expression is blank, because thanks to that, validators will be still composable.
- Let's use `MandatoryParameterValidator` in most places as it was previously. We can add `NotNullParameterValidator` in places where we figure out that not only blank expression is bad, but explicit `null` either.

Reply:
done :)